agent/src/main/java/com/microsoft/applicationinsights/agent/internal/agent/sql/PreparedStatementClassDataProvider.java

```diff
@@ -59,8 +59,10 @@ public void add() {
             factory = classFactoryForMySql();
             doAdd(factory, "com/mysql/jdbc/PreparedStatement");
 
-            factory = classFactoryForPostgreSql();
+            factory = classFactoryForOldPostgreSql();
             doAdd(factory, "org/postgresql/jdbc2/AbstractJdbc2Statement");
+            factory = classFactoryForNewPostgreSql();
+            doAdd(factory, "org/postgresql/jdbc/PgPreparedStatement");
```

Comment:
I would prefer to have these String signatures as constants, in the class. That helps keeping all the signatures at one place in the class. 

I know this is not fully followed in this class, but I would encourage this change.